spec/truncate_html/html_truncator_spec.rb

```diff
@@ -242,4 +242,20 @@ def truncate(html, opts = {})
         to eq '<h1>hello <!-- stuff --> and <!-- la -->...</h1>'
     end
   end
+
+  context 'when truncating by break_token and using a omission' do
```

Comment:
[Minor] `a` -> `an` omission, but I suggest 'when a custom break_token and omission options are used' for consistency with the rest of the file and also discard the first expectation. I think it has already been tested with various `break_token` values above.

Reply:
Done! 
Removed the first expect (indeed it was tested above) and fixed the test description.
Plus I fixed all rubocop warnings in the spec file according to yogurt rubucop.yml